hri_safety_sense/src/main.cpp

```diff
@@ -12,27 +12,24 @@
  * POSSIBILITY OF SUCH DAMAGE.
  */
 
-#include "ros/ros.h"
+#include "rclcpp/rclcpp.hpp"
 #include "VscProcess.h"
 
-hri_safety_sense::VscProcess *VSCInterface;
-
 /**
  * VSC Vehicle Interface
  */
 int main(int argc, char **argv) {
-	ros::init(argc, argv, "VscProcess");
+  rclcpp::init(argc, argv);
 
-	// Create vehicle interface
-	VSCInterface = new hri_safety_sense::VscProcess();
+  rclcpp::NodeOptions node_options;
+  node_options.allow_undeclared_parameters(true);
```

Comment:
Let's not set this unless it is absolutely necessary.  Since you are now declaring all of the parameters, I don't think it is necessary anymore.

Reply:
Done in 45091aa